src/query/expression/src/schema.rs

```diff
@@ -1085,3 +1370,53 @@ pub fn infer_table_schema(data_schema: &DataSchemaRef) -> Result<TableSchemaRef>
     }
     Ok(TableSchemaRefExt::create(fields))
 }
+
+// a complex schema to cover all data types tests.
+pub fn create_test_complex_schema() -> TableSchema {
```

Comment:
This function is just used for testing, better to move to `src/query/expression/tests/it/schema.rs`

Reply:
ColumnLeaf also need this function to generate test data.

Reply:
How about adding a label `#[cfg(test)]`.

Reply:
if label `#[cfg(test)]`, cannot compile.